assets/constants.js

```diff
@@ -19,6 +19,8 @@ var constants = {
     "Mathematics", "Medicine", "Music", "Philosophy", "Physics", "Poetry",
     "Probability", "Programming", "Puzzles", "Reading", "Spanish", "Sport",
     "Statistics", "Trigonometry", "Welcome"],
+  "ACTIVITY_TYPE_EXPLORATION": "exploration",
```

Comment:
Should I let the constant be in feconf.py. It has been used extensively. I'll just send `'exploration'` and `'collection'` from the frontend.